### ISSUE TITLE
Add support for node v18 or make the error message helpful

### ISSUE THREAD

**Issue by iver56** [athombv/apps-sdk-issues#249]
I'm trying to run an app on my homey, but it's failing with an error message `Parameter 'app' is required`. I was able to run apps on my homey without this error last week. What should I do next to narrow down the issue? Also, please consider improving the error message so it becomes more helpful. I've tried googling the error message, and I didn't find a relevant result. I also tried to find the relevant Homey SDK source code so I could dig and find out where the error came from, but I gave up that.

Here are the commands I ran along with the output:

```
(base) C:\Users\Iver\Code\homey-yolo>homey app create
(node:18716) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
? What is your app's name? yolo
? What is your app's description? yolosdf
? What is your app's unique ID? com.iver.yolo
? What platforms will your app support? Homey Pro
? What is your app's category? tools
? Use TypeScript? No
? Use ESLint? No
? Use standard license for Homey Apps (GPL3)? Yes
? Seems good? Yes
✓ Installing dev dependencies: @types/homey@npm:homey-apps-sdk-v3-types, @types/node, @tsconfig/node12
✓ Installation complete
✓ App created in `C:\Users\Iver\Code\homey-yolo\com.iver.yolo`

        Learn more about Homey App development at: https://apps.developer.homey.app

(base) C:\Users\Iver\Code\homey-yolo>cd com.iver.yolo

(base) C:\Users\Iver\Code\homey-yolo\com.iver.yolo>homey app run
(node:1408) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
✓ Pre-processing app...
✓ Validating app...
✓ App validated successfully against level `debug`
✓ Packing Homey App...
 — App archive size: 58 KB, 11 files
✓ Installing Homey App on `Homey Pro` (https://192-168-10-111.homey.homeylocal.com)...
× Parameter 'app' is required
(base) C:\Users\Iver\Code\homey-yolo\com.iver.yolo>node --version
v18.0.0
```

My Homey Pro is running Homey version 7.3.0. I have tried to restart it, and that did not help.

**Comment by WeeJeWel** [comment]
This is new. Can you try with Node.js v16?

**Comment by robertklep** [comment]
Related to the new experimental `fetch` implementation that is turned on by default in Node v18, using `--no-experimental-fetch` fixes the problem.

**Comment by iver56** [comment]
Thanks for the swift response! Yeah, using node v16 does indeed help 👍 I have changed the name of the issue accordingly

**Comment by WeeJeWel** [comment]
Thank you for reporting @iver56 ! This has been patched in `homey@2.28.2`.

You can run `npm install -g homey@2.28.2` and continue using Node.js v18.
